cpp/include/games/nim/Game.hpp

```diff
@@ -0,0 +1,157 @@
+
+#pragma once
+
+#include <array>
+#include <cstdint>
+#include <functional>
+#include <sstream>
+#include <string>
+
+#include <boost/functional/hash.hpp>
+#include <torch/torch.h>
+
+#include <core/BasicTypes.hpp>
+#include <core/concepts/Game.hpp>
+#include <core/GameLog.hpp>
+#include <core/GameTypes.hpp>
+#include <core/SimpleStateHistory.hpp>
+#include <core/TrainingTargets.hpp>
+#include <core/WinShareResults.hpp>
+#include <util/EigenUtil.hpp>
+#include <util/FiniteGroups.hpp>
+#include <util/MetaProgramming.hpp>
+#include <games/nim/Constants.hpp>
+#include <core/TrivialSymmetries.hpp>
```

Comment:
Minor nitpick, but I try to keep the include's in alphabetical order by grouping (system includes, third-party includes, project includes).